lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -143,7 +147,7 @@ export class ManagedIdentityApplication {
                 managedIdentityRequestParams.resource.replace("/.default", ""),
             ],
             authority: this.fakeAuthority.canonicalAuthority,
-            correlationId: this.config.managedIdentityId.id,
+            correlationId: this.correlationId,
```

Comment:
How does a log look like for an MSI token? 

Correlation ID should be per request, not per object.

Reply:
I think I understand. So: the fakeAuthority (needed for fakeClientCredentialClient, which is needed to re-use caching functionality) needs it's own correlationId. The fakeAuthority's correlationId should be different from each network request's correlationId (which will unique for each network request)?

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/6919